src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -1512,6 +1513,12 @@ sap.ui.define([
 			} else {
 				oRequestInfo = this._createRequestInfo(HTTPMethod.DELETE, oBindingInfo.getResourceServerPath());
 				this._setProperty(this.mChangedResources, FHIRUtils.deepClone(aResPath), oRequestInfo, true, sResourceGroupId && sResourceGroupId === sGroupId ? sGroupId : undefined);
+				if (!this.mRemovedResources[oBindingInfo.getResourceType()]) {
+					this.mRemovedResources[oBindingInfo.getResourceType()] = [sResourcePath.substring(1)];
+				} else {
```

Comment:
lets generate a method called `_addToRemovedResources` containing this adding logic here

Reply:
done